h_transport_materials/property_database/palladium.py

```diff
@@ -254,7 +251,237 @@
     isotope="H",
 )
 
-properties = [volkl_diffusivity, favreau_solubility_t, favreau_solubility_h]
+data_inv_T_h = (
+    np.array(
+        [
+            0.003354,
+            0.003354,
+            0.003354,
+            0.003094,
+            0.002679,
+            0.002374,
+            0.002113,
+            0.001744,
+            0.001485,
+            0.001293,
+            0.001293,
+            0.001145,
+            0.001027,
+            0.000852,
+        ]
+    )
+    * htm.ureg.K**-1
+)
+
+data_T_h = 1 / data_inv_T_h
+
+# measured c_s
+data_cs_h = (
+    np.array(
+        [
+            1192.0,
+            797.9,
+            102.7,
+            247.5,
+            350.6,
+            246.6,
+            181.7,
+            124.2,
+            98.8,
+            86.3,
+            85.9,
+            79.0,
+            74.3,
+            70.8,
+        ]
+    )
+    * htm.ureg.mol
+    * htm.ureg.m**-3
+)
+
+# Control pressure
+data_P_h = (
+    np.array(
+        [
+            1343.6,
+            671.6,
+            13.4,
+            134.5,
+            673.3,
+            673.4,
+            673.3,
+            673.3,
+            668.0,
+            668.0,
+            668.0,
+            668.0,
+            668.0,
+            681.3,
+        ]
+    )
+    * htm.ureg.Pa
+)
+
+data_solubility_h = data_cs_h / data_P_h**0.5
+
+solubility_powell_h = htm.Solubility(
+    data_T=data_T_h,
+    data_y=data_solubility_h,
+    isotope="H",
+    note="table I, calculated from measured cs and control pressure columns",
+    source="powell_surface_1991",
+)
+
+
+data_diffusivity_h = (
+    np.array(
+        [
+            3.248e-11,
+            3.339e-11,
+            3.467e-11,
+            7.053e-11,
```

Comment:
This should be 7.056e-11, very minor difference though

Reply:
```suggestion
            7.056e-11,
```